tests/contract/test_resource_client.py

```diff
@@ -543,3 +544,11 @@ def test_has_update_handler(resource_client):
     schema = {"handlers": {"update": {"permissions": ["permission"]}}}
     resource_client._update_schema(schema)
     assert resource_client.has_update_handler()
+
+
+def test_assert_create_time(resource_client):
```

Comment:
can we add negative tests?  You should use `@parametrized` so we test all CRUDL

Reply:
1) I am unsure how we would run a negative test for time constraint. In what condition can we expect a handler to respond in more than 30/60 seconds?
2) Correct me if I am wrong, as far as I know this is being tested for all CRUDL apis right now.

Reply:
i meant for these unit tests

Reply:
My bad. Let me go ahead and add that in.